tests/rest_api/assets/cvat_db/restore.sh

```diff
@@ -0,0 +1,5 @@
+restore="SELECT pg_terminate_backend(pg_stat_activity.pid) FROM pg_stat_activity WHERE pg_stat_activity.datname = 'cvat' AND pid <> pg_backend_pid();\
```

Comment:
It should be replaced by cvat_db.sql with variables (see psql -v)

Reply:
Fixed